masu/config.py

```diff
@@ -84,7 +84,7 @@ class Config:
     REPORT_PROCESSING_BATCH_SIZE = 100000
 
     AWS_DATETIME_STR_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
-    OCP_DATETIME_STR_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+    OCP_DATETIME_STR_FORMAT = '%Y-%m-%d %H:%M:%S +0000 UTC'
```

Comment:
Do we know that this won't clobber valid timezone info?

e.g. If my host node is using UTC+5 for its time zone, will that be honored and passed through to the operator metering reports?

Reply:
This was just the format that the report had.  @adberglund would know more about how it originated.  We have full control over the report so I could see some of this changing as we go along.